src/server/Listener.js

```diff
@@ -140,6 +147,16 @@ class Listener {
 	}
 }
 
+function replaceRequestPlaceholders (req, body) {
+	try {
+		const regex = /\${req\.([\w.]+)}/g;
+		body = _.isString(body) ? body : JSON.stringify(body);
+		return body.replace(regex, (match, props) => _.get(req, props));
```

Comment:
That is probably a good idea. Express will eventually stringify it anyway, but also set content-type to application/json (which we otherwise have to do manually):

https://github.com/expressjs/express/blob/master/lib/response.js#L152
and
https://github.com/expressjs/express/blob/master/lib/response.js#L243-L248